### DISCUSSION TITLE
Which INIs to modify?

### DISCUSSION THREAD

**Discussion by Cyrneco** [X2CommunityCore/xcom2-launcher#303] [Help] [answered]
Which INI files I have to edit for AML to pick on it? The ones on the steam workshop or the ones on the mods folder? 
Brilliant application btw, thanks.

**Comment by RevZero** [comment]
You can use the build-in Editor. Select the desired mod and use the "Config" tab at the bottom mod-info-panel.

![image](https://user-images.githubusercontent.com/936992/152780806-14f1dd03-25a0-4035-8ae3-a661b45c5f15.png)

**Comment by Cyrneco** [comment]
Hi Rev
Thanks. I think I didn't explain myself. I know how to modify ini tru the AML interface, what I don't know is which inis are picked by AML if I have the mods in the steam workshop directory AND the /mods directory. The problem I have is that sometyimes inis changes stop being picked up and I wonder why. It happens especially with LWoTC.

**Comment by RevZero** [comment, marked as answer]
AML will use/edit the ini-files that are located in the config sub-folder in the related mod path (where the mod was found). Check the "Inspect" tab to view the mod folder are select mod right click menu and choose "open in explorer".
